examples/test/text/msdf.html

```diff
@@ -0,0 +1,43 @@
+<html>
+  <head>
+    <title>Text MSDF vs. SDF Font</title>
+    <meta name="description" content="Text MSDF vs. SDF Font - A-Frame">
+    <script src="../../../dist/aframe-master.js"></script>
+  </head>
+  <body>
+    <a-scene>
+      <a-assets>
+        <a-mixin id="text"
+          text="align: center; width: 6;
+               value: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut enim ad minim veniam"
+        ></a-mixin>
+        <a-mixin id="msdf-text"
+          text="align: center; width: 6;
+               font: https://rawgit.com/Jam3/three-bmfont-text/master/test/fnt/Roboto-msdf.json;
+               value: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut enim ad minim veniam"
+        ></a-mixin>
+      </a-assets>
+
+      <a-text mixin="msdf-text" position="-3 0 0" wrap-count="20"></a-text>
+      <a-text mixin="msdf-text" position="-3 2 0" wrap-count="30"></a-text>
+      <a-text mixin="msdf-text" position="-3 3 0" wrap-count="40"></a-text>
+      <a-text mixin="msdf-text" position="-3 3.5 0" wrap-count="50"></a-text>
+      <a-text mixin="msdf-text" position="-3 4 0" wrap-count="60"></a-text>
+      <a-text mixin="msdf-text" position="-3 4.5 0" wrap-count="70"></a-text>
+      <a-text mixin="msdf-text" position="-3 5 0" wrap-count="80"></a-text>
+
+      <a-text mixin="text" font="kelsonsans" position="3 0 0" wrap-count="20"></a-text>
+      <a-text mixin="text" font="kelsonsans" position="3 2 0" wrap-count="30"></a-text>
+      <a-text mixin="text" font="kelsonsans" position="3 3 0" wrap-count="40"></a-text>
+      <a-text mixin="text" font="kelsonsans" position="3 3.5 0" wrap-count="50"></a-text>
+      <a-text mixin="text" font="kelsonsans" position="3 4 0" wrap-count="60"></a-text>
+      <a-text mixin="text" font="kelsonsans" position="3 4.5 0" wrap-count="70"></a-text>
+      <a-text mixin="text" font="kelsonsans" position="3 5 0" wrap-count="80"></a-text>
+
+      <a-sky color="#3CF"></a-sky>
+      <a-entity position="0 2 3.7">
```

Comment:
In our text examples, I'd try to leave the camera at the default 0, 0, 0, and place the content in the desired places instead.